ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-presubmits.yaml

```diff
@@ -81,6 +81,92 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build02
+    context: ci/prow/e2e-console
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      job-release: "4.8"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-openshift-tests-private-master-e2e-console
+    optional: true
```

Comment:
@kuiwang02  Here is the `optional` item.

Reply:
ok